.github/workflows/main.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       - uses: actions/checkout@master
       - uses: actions/setup-node@v3
         with:
-          node-version: 14.x
+          node-version: 18
```

Comment:
Let me know if it causes any issue, I just saw that latest version of [np](https://github.com/sindresorhus/np) requires `node v16`, and figured it would be better to use the latest LTS version, witch is `v18`

Reply:
I think it does not, and it's of great help you looked into it, thanks